frontend/src/MedicalAnnotation.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    popover: {
+      pointerEvents: 'none',
+    },
+    paper: {
+      padding: theme.spacing(1),
+    },
+    annotation: {
+        textDecoration: 'underline',
+        fontSize: "30pt"
+    }
+  }),
+);
+
+export interface AnnotationEntity {
+    Id: number,
+    BeginOffset: number,
+    EndOffset: number,
+    Score: number,
+    Text: string,
+    Category: string,
+    Type: string
+}
+
+interface MedicalAnnotationProps {
+    entity: AnnotationEntity
+}
+
+export function MedicalAnnotation(props: MedicalAnnotationProps) {
+    const classes = useStyles();
+    const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
+
+    const handlePopoverOpen = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    
+    const handlePopoverClose = () => {
+        setAnchorEl(null);
+    };
+
+    const open = Boolean(anchorEl);
```

Comment:
You can just write:

    const open = !!anchorEl;

Reply:
I don't understand why that is better.

Reply:
Taste I suppose.  It looks simpler to me.